daemon/openrazer_daemon/hardware/device_base.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, device_path, device_number, config, persistence, testing, add
         self._observer_list = []
         self._effect_sync_propagate_up = False
         self._disable_notifications = False
-        self._disable_persistence = False
+        self._disable_persistence = config.getboolean('Startup', "restore_persistence") is not True
```

Comment:
This line doesn't make too much sense to me right now, `disable_persistence` is only used to avoid writing to persistence from suspend_device & resume_device, so where on purpose it's temporarily not writing anything there.

If you set this during init, yes it won't write to persistence until you lock your screen (when suspend_device is called), so whatever you want this to do, it seems like a misuse of what it's currently doing?

Reply:
I suggested this change to avoid having `config.getboolean('Startup', "restore_persistence")` in multiple places. I did suspect it would need looking at more carefully as it was used for that suspend/resume logic. `= False` and `= True` will overwrite this state.

Currently, the code will always be writing persistence, irrespective of user preference, but you're right, the configuration is for **restoring**, not _enabling_ persistence.